src/Microsoft.DocAsCode.Build.Common/ApplyOverwriteDocument.cs

```diff
@@ -14,10 +14,10 @@ namespace Microsoft.DocAsCode.Build.Common
 
     public abstract class ApplyOverwriteDocument : BaseDocumentBuildStep
     {
-        private readonly MergerFacade Merger = new MergerFacade(
-                new DictionaryMerger(
-                    new KeyedListMerger(
-                        new ReflectionEntityMerger())));
+        protected virtual MergerFacade Merger { get; set; } = new MergerFacade(
```

Comment:
Add todo and consider to add later

---
In reply to: [91022771](https://github.com/dotnet/docfx/pull/965#discussion_r91022771) [](ancestors = 91022771)